Libraries/plist/Sources/Format/BaseXMLParser.cpp

```diff
@@ -9,21 +9,284 @@
 
 #include <plist/Format/BaseXMLParser.h>
 
+#include <algorithm>
+#include <mutex>
 #include <cerrno>
 #include <cstdlib>
+#include <cstdarg>
+
+#if _WIN32
+#include <windows.h>
+#endif
 
 using plist::Format::BaseXMLParser;
 
+#if _WIN32
+using WideString = std::basic_string<std::remove_const<std::remove_pointer<LPCWSTR>::type>::type>;
```

Comment:
Yeah I suppose I don't disagree with the `remove_pointer` and `remove_const` stuff. If you want to keep this this way I'm fine with it.